config/database.php

```diff
@@ -28,6 +28,35 @@
 		'charset'      => 'utf8',
 		'caching'      => FALSE,
 	),
+	'alternate_' => array
```

Comment:
Can we add this as a commented out example rather than adding a new `alternate_` config group?

* We don't want to encourage end-users to actually use the `alternate_` connection to extend our default config
* In the unlikely case a user already has an `alternate_` db, we don't want to pollute that config

I know that's different to how the other groups appear, but it is consistent with what we ended up doing in kohana/cache so that the module just ships with example config but it's for end users to copy/paste/configure 
their own connections however they want them.